tools/nni_cmd/nnictl.py

```diff
@@ -84,6 +84,10 @@ def parse_args():
     parser_updater_trialnum.add_argument('--id', '-i', dest='id', help='the id of experiment')
     parser_updater_trialnum.add_argument('--value', '-v', required=True)
     parser_updater_trialnum.set_defaults(func=update_trialnum)
+    parser_updater_feed_tuning_data = parser_updater_subparsers.add_parser('feed', help='feed additional data for tuning')
+    parser_updater_feed_tuning_data.add_argument('id', nargs='?', help='the id of experiment')
+    parser_updater_feed_tuning_data.add_argument('--filename', '-f', required=True)
+    parser_updater_feed_tuning_data.set_defaults(func=update_feed_tuning_data)
```

Comment:
It's too long, can it be a little shorter or use shorthand?

Reply:
Any other good suggestions on the premise of guaranteeing semantics?

Reply:
"update feed tuning data" doesn't meet the grammar specification also confuse user, I think it will be better to use "update tuning data" or something else :)

Reply:
Modify it to a root comman `nnictl feed`